### PR TITLE
feat/player-name-version: support configurable player name and versio…

### PR DESCRIPTION
**Pull request by cjpillsbury**
…n based on current architecture of mux-player and mux-player-react.

### CODE REVIEW

File: packages/mux-player/src/index.ts [hunk 1]
```diff
@@ -352,11 +356,15 @@ class MuxPlayerElement extends VideoApiElement {
   }
 
   get playerSoftwareName() {
-    return playerSoftwareName;
+    return this.hasAttribute(MuxVideoAttributes.PLAYER_SOFTWARE_NAME)
+      ? this.getAttribute(MuxVideoAttributes.PLAYER_SOFTWARE_NAME)
+      : playerSoftwareName;
```
Comment (luwes) [inline packages/mux-player/src/index.ts:361]:
```suggestion
    return this.getAttribute(MuxVideoAttributes.PLAYER_SOFTWARE_NAME) ?? playerSoftwareName;
```

Comment (luwes) [reply packages/mux-player/src/index.ts:361]: a bit less code, works for `null` or `undefined`

**Comment by luwes** [comment]
looks good, one non-blocking nitpick, one code conflict to fix

**Comment by dylanjha** [comment]
QA pass LGTM

mp4_2022-02-22_22-59-08
mpw_2022-02-22_22-59-32

**Pull request merged**